src/resources/cdm-detailed.properties

```diff
@@ -94,27 +94,33 @@ spark.cdm.connect.target.password      cassandra
 #                           the WRITETIME of the target record.
 #
 # Other Parameters:
-#  spark.cdm.schema.origin
-#    .column
+#  spark.cdm.schema
+#    .origin.column
 #       .names.to.target  : Default is empty. If column names are changed between Origin and Target, then
 #                           this map-like list provides a mechanism to associate the two. The format is
 #                           origin_column_name:target_column_name. The list is comma-separated. Only renamed
 #                           columns need to be listed.
+#
+#    .ttlwritetime.calc
+#       .useCollections   : Default is false. When true, TTL and WRITETIME max calculations will include
+#                           collections and UDTs. This is useful when the only non-PK columns are collections
+#                           and/or UDTs.
 #-----------------------------------------------------------------------------------------------------------
 spark.cdm.schema.origin.keyspaceTable                keyspace_name.table_name
 
 # Max TTL value of all non-PK columns will be used for insert on target
-#spark.cdm.schema.origin.column.ttl.automatic         true
+spark.cdm.schema.origin.column.ttl.automatic         true
 
 # Max TTL value of specified non-PK columns will be used for insert on target (overrides automatic setting)
 #spark.cdm.schema.origin.column.ttl.names             data_col1,data_col2,...
 
 # Max WRITETIME value of all non-PK columns will be used for insert on target
-#spark.cdm.schema.origin.column.writetime.automatic   true
+spark.cdm.schema.origin.column.writetime.automatic   true
 
 # Max WRITETIME value of specified non-PK columns will be used for insert on target  (overrides automatic setting)
 #spark.cdm.schema.origin.column.writetime.names       data_col1,data_col2,...
 #spark.cdm.schema.origin.column.names.to.target       partition_col1:partition_col_1,partition_col2:partition_col_2,...
+spark.cdm.schema.ttlwritetime.calc.useCollections    false
```

Comment:
nitpick - can we get rid of the camelCase here?

Reply:
All multi-word phrases (keyspaceTable, runId, numParts, batchSize, splitRegex, etc.) are currently use camel case, so sticking with that standard.